tests/functional/pages/surveys/census/household/household-composition.page.js

```diff
@@ -0,0 +1,38 @@
+import QuestionPage from '../../question.page'
+
+class HouseholdCompositionPage extends QuestionPage {
```

Comment:
Not sure what has been agreed. I have reused them where possible. I guess generally the benefit of the page models are that they are reusable bits of logic